src/hardware/pcspeaker_discrete.cpp

```diff
@@ -341,7 +338,7 @@ void PcSpeakerDiscrete::SetType(const PpiPortB &b)
 }
 
 // Halt the channel after the speaker has idled
-void PcSpeakerDiscrete::PlayOrSleep(const uint16_t speaker_movements,
+void PcSpeakerDiscrete::PlayOrSleep(const bool speaker_movements,
```

Comment:
Just by looking at the name, I only have a very vague idea about what "speaker_movements" signifies. Could you find a better name perhaps?

Reply:
Yes - now "samples_were_processed".

The good news is this all disappears when we move to auto-sleep.